modules/editorial-metadata/lib/components/editorial-metadata-manager.js

```diff
@@ -0,0 +1,181 @@
+import apiFetch from '@wordpress/api-fetch';
+import {
+	Button,
+	Card,
+	CardHeader,
+	Flex,
+	FlexItem,
+	__experimentalHeading as Heading,
+	Notice,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { useState } from '@wordpress/element';
+import { __, sprintf } from '@wordpress/i18n';
+
+import CreateEditEditorialMetadataModal from './modals/create-edit-editorial-metadata-modal';
+import ConfirmDeleteModal from '../../../shared/js/components/modals/confirm-delete-modal';
+import SuccessNotice from '../../../shared/js/components/success-notice';
+
+export default function EditorialMetadataManager( {
+	supportedMetadataTypes,
+	editorialMetadataTerms,
+} ) {
+	const [ success, setSuccess ] = useState( null );
+	const [ error, setError ] = useState( null );
+	const [ availableMetadataTypes ] = useState( supportedMetadataTypes );
+	const [ eMetadataTerms, setEMetadataTerms ] = useState( editorialMetadataTerms );
+	const [ eMetadataTerm, setEMetadataTerm ] = useState( null );
+	const [ isConfirmingDelete, setIsConfirmingDelete ] = useState( false );
+	const [ isModalVisible, setIsModalVisible ] = useState( false );
+
+	const handleErrorThrown = error => {
+		setSuccess( null );
+		setError( error );
+		setIsModalVisible( false );
+		setIsConfirmingDelete( false );
+		setEMetadataTerm( null );
+	};
+
+	const handleSuccess = ( message, eMetadataTermResult ) => {
+		setError( null );
+		setSuccess( message );
+		if ( eMetadataTerm && ! isConfirmingDelete ) {
```

Comment:
Minor: These lines seem a little close together to me. Could you add some whitespace here and after the final `else` close bracket?